### PR TITLE
Contact History Tweaks

### PR DESCRIPTION
**Pull request by ericnewcomer**
Some tweaks to the contact history:
* Use the last 100 messages to determine the window for events. 
* Give feedback on initial load.
* Don't show more button if there aren't any more messages

One unfortunate side effect is that this will hide any events that occurred before the first message and will also not surface events that have occurred since the last message.

The bonus is that its a far nicer user experience for the vast majority of cases.

### CODE REVIEW

**Comment by nicpottier** [comment]
Why the side effect of no events before the first message? Seems we should be able to know both those cases and adjust the windows appropriately no?

**Comment by ericnewcomer** [comment]
Ya, actually I think there's a clever way to do it without issuing more queries come to think of it.

**Comment by ericnewcomer** [comment]
Updated to include:

* Events that happen before or after the given the window derived from message list
* Remove the date at the bottom (since it's always the same as the last item in the list)
* Refresh the last 7 days worth of history on a 30s interval, trigger immediate refresh on message send
* More tests

Ready for another looksie.

File: templates/msgs/msg_send_modal.haml [hunk 1]
```diff
@@ -78,7 +78,7 @@
     }
 
     
-    function showCompose(loading, url) {
+    function showCompose(loading, url, onComplete) {
```
Comment (nicpottier) [inline templates/msgs/msg_send_modal.haml]: Do we need to change other calls to showCompose (like on the msg inbox page?)

Comment (ericnewcomer) [inline templates/msgs/msg_send_modal.haml]: Nope, parameters are optional. I did test sending from the inbox page manually to make sure I didn't break that.

Comment (nicpottier) [inline templates/msgs/msg_send_modal.haml]: excellent. ok, let's ship it.

**Pull request merged**